src/app/js/components/column/messageSender/MessageSenderView.js

```diff
@@ -101,7 +106,7 @@ class MessageSender extends Component {
             return
 
         if ( this.state.eventName && this.state.messageCollection.length )
-            this.props.sendMessage(this.props.activeTab, this.state.eventName, this.state.messageCollection)
+            this.props.sendMessage(this.props.activeTab, this.state.eventName, this.state.messageCollection,this.state.messageIsJsonCollection)
```

Comment:
The 'sendMessage' function doesn't accept this parameter. This also needs to be added to the function that's mapped to the props in 'MessageSender.js'.